### ISSUE TITLE
accessing session variable from SQL functions

### ISSUE THREAD

**Issue by lionelB** [hasura/graphql-engine#4594]
Hello, 

I'm currently reading the documentation in order to use Hasura. I've encounter 2 error for now 
- https://hasura.io/docs/1.0/graphql/manual/schema/computed-fields.html#accessing-hasura-session-variables-in-computed-fields

It's not clear how to make it work using the hasura console webapp. 
using web developer network tool, it seems that adding a computed field  don't add 
this line
```diff 
{
    "type":"add_computed_field",
    "args":{
        "table":{
            "name":"article",
            "schema":"public"
        },
        "name":"liked_by_user",
        "definition":{
            "function":{
                "name":"article_liked_by_user",
                "schema":"public"
            },
            "table_argument":"article_row",
-            "session_argument":"hasura_session"
        }
    }
}
```
which results in the following error

```json
{
  "errors": [
    {
      "extensions": {
        "path": "$.selectionSet.data_article.selectionSet.liked_by_user.args",
        "code": "validation-failed"
      },
      "message": "the required argument \"args\" is missing"
    }
  ]
}
```

I've also the above error while following this part of the documentation
https://hasura.io/docs/1.0/graphql/manual/schema/custom-functions.html#accessing-hasura-session-variables-in-custom-functions

**Comment by lionelB** [comment]
I've also try to make a post request "by hand" using developper tools to add the missing "session_argument". post request succeed but same error in qraphql

**Comment by rikinsk** [comment]
@lionelB This seems to be a case of docs getting preimptively deployed. This feature is currently not yet released but should be out in the next release.

**Comment by 0x777** [comment]
@lionelB About the GraphQL error that you see, you'll need to add `args: {}` in your graphql query. Ideally `args` should be optional if there are no required arguments, we'll improve this.

**Comment by lionelB** [comment]
thanks !

**Comment by rikinsk** [comment]
closed via #4606
